README.md

```diff
@@ -94,10 +96,23 @@ $ ./do delete:docker      # stop and remove all running docker containers.
 $ ./do qa:lint             # PHP code linter.
 $ ./do qa:lint:javascript  # JS code linter.
 $ ./do qa                  # PHP and JS linters.
+
+$ ./do container:dump      # Dumps DI container.
 ```
 
 # Coding and Testing
 
+## DI
+
+We use Symfony/dependency-injection container. Container configuration can be found in `libs/DI/ContainerFactory.php`
+The container is configured and used with minimum sub-dependencies to keep final package size small. 
+You can check [the docs](https://symfony.com/doc/3.4/components/dependency_injection.html) to learn more about Symfony Container.
+
+## Mozart
+
+We use Mozart plugin for composer to prevent plugin libraries conflicts in PHP. Mozart prefix dependencies namespaces and moves them into `libs\Dependencies` directory.
```

Comment:
Should we also mention how could the conflicts happen? Something about multiple WP plugins using the same composer dependency with different versions?